notebooks/Tutorial4_Client_side_processors.py

```diff
@@ -258,7 +265,9 @@
 dataset = client.pandas_integration.upload_dataframe(df2)
 job = client.jobs.create_avatarization_job(
     AvatarizationJobCreate(
-        parameters=AvatarizationParameters(k=20, ncp=2, dataset_id=dataset.id)
+        parameters=AvatarizationParameters(
+            k=20, ncp=2, dataset_id=dataset.id, seed=1234
```

Comment:
why do we add a seed ? it is probably better not to show users examples with a set seed because it is bad practice